docs/getting-started-guides/docker-multinode.md

```diff
@@ -53,7 +49,9 @@ Please install Docker 1.6.2 or Docker 1.7.1.
 
 ## Prerequisites
 
-1. You need a machine with docker of right version installed.
+The only thing you need is a machine with **Docker 1.8.x or higher**
+
+It might work with older docker versions, but that isn't supported.
```

Comment:
I'm sorry but is this strictly required? I'm still working on docker 1.7.1 and it fits me well.
Maybe it's worth keeping the issue thing:-)